pepper-apis/config/DataDonationPlatform.yaml.ctmpl

```diff
@@ -0,0 +1,29 @@
+{{with $environment := env "ENVIRONMENT"}}
+{{with $gae := env "GAE"}}
+service: pepper-backend
+runtime: java11
+instance_class: B8
+{{if eq $environment "prod"}}
+basic_scaling:
+  max_instances: 6
+  idle_timeout: 45m
+{{else}}
+basic_scaling:
+  max_instances: 4
+  idle_timeout: 20m
+{{end}}
+
+network:
+  instance_tag: study-server # necessary for firewall rules so that DSM will accept https requests from pepper
+{{if and (eq $environment "dev") (eq $gae "true") }}
+entrypoint: java -javaagent:tcell/tcellagent.jar -agentpath:/opt/cprof/profiler_java_agent.so=-cprof_enable_heap_sampling=true,-cprof_service=pepper-backend,-cprof_cpu_use_per_thread_timers=true -Xmx1640m -Xmx1640m -jar DataDonationPlatform.jar
+{{else}}
+entrypoint: java -javaagent:tcell/tcellagent.jar -Xmx1640m -Xmx1640m -jar DataDonationPlatform.jar
```

Comment:
Noted that I added the agent to collect profiler data. I am inclined to make this change permanent for dev deployments until we have a better profiling option.